oktapam/convert/resource_checkout_settings.go

```diff
@@ -3,6 +3,11 @@ package convert
 import (
```

Comment:
I'm not particularly enamoured with the package name `/convert/`

Reply:
Considering the purpose of converting files, renaming it to `modelconv` would make it more concise and descriptive?

Reply:
Hmm... Looking at the AWS provider, they call the processes Expand and Flatten, and the process of doing that "flexing", so I'm kinda leaning towards that.